VideoWeb/VideoWeb/Controllers/VenuesController.cs

```diff
@@ -18,33 +21,35 @@ public class VenuesController : Controller
     {
         private readonly IVideoApiClient _videoApiClient;
         private readonly ILogger<VenuesController> _logger;
+        private readonly IBookingsApiClient _bookingsApiClient;
 
-        public VenuesController(IVideoApiClient videoApiClient, ILogger<VenuesController> logger)
+
+        public VenuesController(IVideoApiClient videoApiClient, ILogger<VenuesController> logger, IBookingsApiClient bookingsApiClient)
         {
             _videoApiClient = videoApiClient;
             _logger = logger;
+            _bookingsApiClient = bookingsApiClient;
         }
 
         /// <summary>
-        /// Get Judge names
+        ///     Get available courts
         /// </summary>
-        /// <returns>List of judges with hearing scheduled, if any</returns>
-        [HttpGet]
-        [ProducesResponseType(typeof(JudgeNameListResponse), (int)HttpStatusCode.OK)]
-        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
-        [SwaggerOperation(OperationId = "GetDistinctJudgeNames")]
-        public async Task<ActionResult<JudgeNameListResponse>> GetDistinctJudgeNamesAsync()
+        /// <returns>List of courts</returns>
+        [HttpGet("courts", Name = "GetCourts")]
```

Comment:
Maybe rename GetCourts to GetVenues or vice versa so it is consistent across the swagger operation and logs.